gremlin-groovy/src/test/java/org/apache/tinkerpop/gremlin/groovy/jsr223/GremlinScriptCheckerTest.java

```diff
@@ -53,6 +104,14 @@ public void shouldIdentifyTimeoutAsStringKeySingleQuoted() {
                 getTimeout().get().longValue());
     }
 
+    @Test
+    public void shouldIdentifyRequestIdAsStringKeySingleQuoted() {
+        assertEquals("db024fca-ed15-4375-95de-4c6106aef895", GremlinScriptChecker.parse("g.with('requestId', 'db024fca-ed15-4375-95de-4c6106aef895').with(true).V().out('knows')").
```

Comment:
is uppercase UUID supported?

Reply:
yes - should pick up any string really. modified one of the tests to do uppercase uuid though to just make the test more realistic to what people would submit.